hw2/01-coins.js

```diff
@@ -1,7 +1,131 @@
-/** Exercise 01 - Coins **/
+/* eslint-disable no-console */
+/** Exercise 01 - Coins * */
 
+/**
+ * Removes as many dollars as possible and returns
+ * amount removed and amount remaing
+ */
+const removeDollar = (num) => {
+  let dollars = 0;
+  let rem = num;
+  while (rem > 100) {
```

Comment:
The method you used to extract the amount of the different coins from the user's input is great. I liked your use of the while loop and subtracting until you have no more dollars to subtract from the value.